pkg/sdn/plugin/subnets.go

```diff
@@ -220,7 +221,15 @@ func (master *OsdnMaster) watchSubnets() {
 					log.Errorf("Error in deleting annotated subnet from master, name: %s, ip %s: %v", name, hostIP, err)
 					return nil
 				}
-				err = master.addNode(name, hostIP)
+				var hsAnnotations map[string]string
+				if vnid, ok := hs.Annotations[osapi.FixedVnidHost]; ok {
```

Comment:
I'd say it's fine to do this in both multitenant and single-tenant. Especially if we only support 0 for the fixed vnid.